### PR TITLE
feat: add enterprise associations info

### PR DESCRIPTION
**Pull request by long74100**
https://openedx.atlassian.net/browse/ENT-5251

- Add table to show all enterprise associations for a user under account information
- misc fixes

Screen Shot 2022-01-05 at 6 11 09 PM

Screen Shot 2022-01-05 at 6 09 54 PM

### CODE REVIEW

File: src/components/Table.jsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default function Table({
           {rows.map((row) => {
             prepareRow(row);
             return (
-              <React.Fragment {...row.key}>
+              <React.Fragment key={row.id}>
```
Comment (manny-m) [inline src/components/Table.jsx:55]: 🚀

File: src/users/data/api.js [hunk 2]
```diff
@@ -140,6 +140,21 @@ export async function getUser(userIdentifier) {
...
 
+export async function getEnterpriseCustomerUsers(username, page = 1) {
+  const baseUrl = AppUrls.getEnterpriseCustomerUsersUrl(username);
+  const queryString = `username=${username}&page=${page}`;
+  const { data } = await getAuthenticatedHttpClient().get(
+    `${baseUrl}?${queryString}`,
+  );
+
+  if (data.next) {
+    const nextPageEnterpriseCustomerUsers = await getEnterpriseCustomerUsers(username, data.current_page + 1);
```
Comment (manny-m) [inline src/users/data/api.js:149]: I love to see it ♻️

**Review by manny-m — APPROVED** [review]
LGTM 👍

File: src/users/data/api.js [hunk 3]
```diff
@@ -140,6 +140,21 @@ export async function getUser(userIdentifier) {
   }
 }
 
+export async function getEnterpriseCustomerUsers(username, page = 1) {
+  const baseUrl = AppUrls.getEnterpriseCustomerUsersUrl(username);
+  const queryString = `username=${username}&page=${page}`;
```
Comment (adamstankiewicz) [inline src/users/data/api.js]: nit: could use `URLSearchParams()` to generate the query string, e.g.:

```js
const queryParams = new URLSearchParams({ username, page });
const { data } = await getAuthenticatedHttpClient().get(`${baseUrl}?${queryParams.toString()}`
```

**Review by adamstankiewicz — APPROVED** [review]
LGTM! Just one optional nit.

**Pull request merged**